bundles/aero.minova.rcp.model/src/aero/minova/rcp/model/QuantityValue.java

```diff
@@ -23,6 +20,31 @@ public String getUnit() {
 
 	@Override
 	public String toString() {
-		return MessageFormat.format("QuantityValue [type=DOUBLE, value={0},unit={1}]", number, unit);
+		return MessageFormat.format("QuantityValue [type=" + dataType.toString() +  ", value={0},unit={1}]", String.valueOf(getValue()), unit);
+	}
+	
+	/**
+	 * Diese Methode liefert ein Object zurück, für den übergebenen String.
+	 * 
+	 * @param text
+	 *            Wert aus dem das VAlue gebildet werden soll
+	 * @param negative
+	 *            true - negative Zahl
+	 * @param type
+	 *            DataType des Fields
+	 * @param dfs
+	 *            DecimalFormatSymbols
+	 * @return Value für den entsprechenden DataType
+	 */
+	public static Object getNumberObjectFromString(String text, DataType type, DecimalFormatSymbols dfs) {
```

Comment:
Genau diese Methode gibt es in der Klasse `NumberFormatUtil` schon.

Dort außerdem `Number` statt `Object` benutzen.

Reply:
Kann auf die Util Klasse aber nicht zugreifen, da es sonst zu Cycle Abhängigkeiten kommen würde

Reply:
Dann schmeiß es aus der `NumberFormatUtil` Klasse raus oder (noch besser) verschiebe die `NumberFormatUtil` Klasse ins Projekt `aero.minova.rcp.constant` ins Package `aero.minova.rcp.util` oder in Projekt `aero.minova.rcp.model` Package `aero.minova.rcp.model.util` wenn das geht.

Doppelten Code sollten wir auf jeden Fall vermeiden.